apps/tailwind-components/components/input/Label.vue

```diff
@@ -21,15 +18,15 @@ withDefaults(
 
 <template>
   <label
-    :for="for"
```

Comment:
we need the 'for' for the ARIA right?

Reply:
do I understand it doesn't need a prop (which makes sense but might be better to have it as prop to help us to comply?)

Reply:
It's for ensuring we follow good ARIA and semantic HTML practices. I put it as a required prop so that typescript will throw an error if this is missing.

Reply:
great